Sources/Commands/SwiftTool.swift

```diff
@@ -117,11 +117,11 @@ private class ToolWorkspaceDelegate: WorkspaceDelegate {
     }
 
     func willUpdateRepository(package: PackageIdentity, repository url: String) {
-        self.outputHandler("Updating \(url)", false)
+        self.outputHandler("Checking for updates from \(url)", false)
     }
 
     func didUpdateRepository(package: PackageIdentity, repository url: String, duration: DispatchTimeInterval) {
-        self.outputHandler("Updated \(url) (\(duration.descriptionInSeconds))", false)
+        self.outputHandler("Checked for updates from \(url) (\(duration.descriptionInSeconds))", false)
```

Comment:
I think this should remain "Updated", as we may have updated. We could / should modify this delegate  to include the modified: Bool argument, but not sure how easy or hard that would be